index.bs

```diff
@@ -179,8 +190,65 @@ extensible.
 </script>
 ```
 
+## Display Arbitrary Content ## {#example-arbitrary-content}
+
+```html
+<div style="display: none;">
+  <div id="interactive-ui"></div>
+  <div id="non-interactive-ui"></div>
+</div>
+
+<video id="video" src="https://example.com/file.mp4"></video>
+
+<button id="pipButton"></button>
+
+<script>
+  const video = document.getElementById('video');
+  const pipButton = document.getElementById('pipButton');
+  const interactiveUI = document.getElementById('interactive-ui');
+  const nonInteractiveUI = document.getElementById('non-interactive-ui');
+
+  pipButton.addEventListener('click', () => {
```

Comment:
How would the website PiP content if there is no button to do so?

Reply:
I think Francois' point is that a website may want to only have interactive Picture-in-Picture and they would want to do early detection. I think that's a fair point and maybe it would require adding something to the API? I would suggest to open an issue about this instead of blocking this PR though.

Reply:
Ah I see. I have filed https://github.com/WICG/picture-in-picture/issues/139